modules/tensor_mechanics/src/kernels/InertialForceBeam.C

```diff
@@ -80,6 +82,7 @@ InertialForceBeam::InertialForceBeam(const InputParameters & parameters)
     _area(coupledValue("area")),
     _Ay(coupledValue("Ay")),
     _Az(coupledValue("Az")),
+    _Ix(isParamValid("Ix") ? coupledValue("Ix") : _zero),
```

Comment:
This is good, but I noticed that you are checking for the validity of the other parameters even though you are using them to get coupling components. This is a bad idea. You should either make them required, provide a default (my favorite), or check each of them before accessing their values.